js/components/portal-dashboard/feedback/rubric-table.tsx

```diff
@@ -0,0 +1,155 @@
+import React from "react";
+// import { Map } from "immutable";
+import Markdown from "markdown-to-jsx";
+import LaunchIcon from "../../../../img/svg-icons/launch-icon.svg";
+
+import css from "../../../../css/portal-dashboard/feedback/rubric-table.less";
+interface IProps {
+  activityStarted: boolean;
+  rubric: any;
+  student: any;
+  rubricFeedback: any;
+  activityId: string;
+  activityIndex: number;
+  updateActivityFeedback?: (activityId: string, activityIndex: number, platformStudentId: string, feedback: any) => void;
+}
+export class RubricTableContainer extends React.PureComponent<IProps> {
+  render() {
+    const { activityStarted, rubric } = this.props;
+    const { criteria } = rubric;
+
+    if (activityStarted) {
+      return (
+        <div className={css.rubricContainer} data-cy="rubric-table">
+          {this.renderColumnHeaders(rubric)}
+          <div className={css.rubricTable}>
+            {criteria.map((crit: any) =>
+              <div className={css.rubricTableRow} key={crit.id} id={crit.id}>
+                <div className={css.rubricDescription}>
+                  <Markdown>{crit.description}</Markdown>
+                </div>
+                {this.renderRatings(crit)}
+              </div>
+            )}
+          </div>
+        </div>
+      );
+    }
+
+    return null;
```

Comment:
I would write this as ternary instead of using 2 returns.  But looking more closely at it, I think the better option is to simply not render this component in the parent if activityStarted is false.  So remove activityStarted from the props, take out the if(activityStarted) and take out the return null.  Then simply have the parent say:
`{ activityStarted && <RubricTable/> }`